resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
+cur=$(head -1 $restartfile)
+initial=$cur
+timeout="300"
+source /etc/sysconfig/puppetserver
+
+if [ -e $restartfile ];  then
+    /usr/bin/java $JAVA_ARGS -Djava.security.egd=/dev/urandom -cp "${INSTALL_DIR}/puppet-server-release.jar" clojure.main -m puppetlabs.trapperkeeper.main --config "${CONFIG}" -b "${BOOTSTRAP_CONFIG}" &
```

Comment:
This comment from the other PR here https://github.com/puppetlabs/ezbake/pull/323#discussion_r72710963 makes me think I should back away from making more comments until the reload PR has had some eyes on it.